src/eslint-config-adeira/ourRules.js

```diff
@@ -333,6 +333,7 @@ module.exports = {
   'jest/valid-expect': ERROR,
   'jest/valid-expect-in-promise': ERROR,
   'jest/valid-title': ERROR,
+  'jest/no-interpolation-in-snapshots': NEXT_VERSION_ERROR,
```

Comment:
@tbergq Could you please update a snapshot? Not sure if that's the only thing which is missing or we should add more.